administrator/components/com_patchtester/PatchTester/View/Pulls/tmpl/default_items.php

```diff
@@ -40,14 +40,14 @@
            </div>
          <div class="row">
               <div class="col-md-auto">
-                    <a class="badge bg-info" href="<?php echo $item->pull_url; ?>" target="_blank">
+                    <a class="badge btn-info" href="<?php echo $item->pull_url; ?>" target="_blank">
                         <?php echo Text::_('COM_PATCHTESTER_VIEW_ON_GITHUB'); ?>
                  </a>
                </div>
                 <?php if ($this->trackerAlias) :
                     ?>
              <div class="col-md-auto">
-                  <a class="badge bg-info"
+                  <a class="badge btn-info btn-sm"
```

Comment:
@N6REJ The `btn-sm` class has no effect because the CSS rules are overridden by the `badge` class. Is there any reason you want to make this button smaller?

Reply:
originally I was using btn class.  Your right badge overides it.. I'll fix it now.

Reply:
fixed